themes/origins/layouts/theme.liquid

```diff
@@ -1,12 +1,22 @@
 <!doctype html>
 <html lang="en" dir="ltr">
   <head>
+    {% style %}
+      :root {
+        --font-heading-scale: {{ settings.heading_scale | times: 1.0 | divided_by: settings.body_scale }};
+        --font-body-scale: {{ settings.body_scale | divided_by: 100.0 }};
+      }
+    {% endstyle %}
+
```

Comment:
It would be nice to include those variables in the `head-config` file

Reply:
True, good idea.